### PR TITLE
make life easier for moderators

### PR DESCRIPTION
**Pull request by stonedDiscord**
allow them to unlock areas
cm themselves
change the motd

### CODE REVIEW

File: server/commands/admin.py [hunk 1]
```diff
@@ -34,9 +34,14 @@ def ooc_cmd_motd(client, arg):
     Show the message of the day.
     Usage: /motd
     """
-    if len(arg) != 0:
-        raise ArgumentError("This command doesn't take any arguments")
-    client.send_motd()
+    if arg == '':
+        client.send_motd()
+    elif client.is_mod:
+        client.server.config['motd'] = arg
```
Comment (oldmud0) [inline server/commands/admin.py]: how to make newlines
also wouldn't it be better to just fix /refresh

File: server/commands/casing.py [hunk 2]
```diff
@@ -172,6 +173,18 @@ def ooc_cmd_uncm(client, arg):
             client.send_ooc(
                 f'{id} does not look like a valid ID.')
 
+@mod_only(area_owners=True)
```
Comment (oldmud0) [inline server/commands/casing.py]: does this mean you can usurp other cms in the same area - e.g. non-mod cm calls /clear_cm to un-cm a mod

Comment (stonedDiscord) [reply server/commands/casing.py]: yes but they can already do that with /uncm

File: server/commands/admin.py (hunk 1 above)
Comment (stonedDiscord) [reply server/commands/admin.py]: refresh does load the MOTD correctly but maybe i don't want to change the config file every time

**Pull request merged**